tests/hpc/install.pm

```diff
@@ -18,7 +18,6 @@
 use base "opensusebasetest";
 use strict;
 use testapi;
-use utils qw/wait_boot/;
 
 sub run() {
     my $self = shift;
```

Comment:
yes, I will do that consistently in another commit changing every appearance after another PR ensuring a consistent code style for all code.